nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/service/canvas-context-menu.service.ts

```diff
@@ -740,8 +741,15 @@ export class CanvasContextMenu implements ContextMenuDefinitionProvider {
                 },
                 clazz: 'fa fa-refresh',
                 text: 'Refresh remote',
-                action: () => {
-                    // TODO - refreshRemoteFlow
+                action: (selection: any) => {
+                    const d = selection.datum();
```

Comment:
The condition for both this action and the condition for `Go To` need to be updated to also check `this.canvasUtils.canRead(selection)`.